### DISCUSSION TITLE
Fix bugs reported by users

### DISCUSSION THREAD

**Discussion by trvqhuy** [trvqhuy/ha-evn#27] [Announcements]
- [x ] Fixed Attribute Error ('ConfigEntries' object has no attribute 'async_setup_platforms' )

<hr /><em>This discussion was created from the release <a href='https://github.com/trvqhuy/nestup_evn/releases/tag/v2.2.0'>Fix bugs reported by users</a>.</em>

**Comment by christianonpele** [comment]
thank you so much for quick response and fixing.

**Comment by Anhnq09** [comment]
i got an issue "Unexpected error. Please contact github@trvqhuy."
